### PR TITLE
Update index.php

### PR DESCRIPTION
**Pull request by dilipchauhan14**
Removed Useless Lines.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -3,10 +3,6 @@
 aws api endpoint check
 fvf
 <?php
-//good boy
```
Comment (dilipchauhan14) [inline index.php]: Following four lines were useless.

File: index.php [hunk 2]
```diff
@@ -238,7 +234,7 @@ function test_input($data) {
 	</div>
 	</form>
 
-ooooo no man
```
Comment (dilipchauhan14) [inline index.php]: Another Useless Line

File: index.php [hunk 3]
```diff
@@ -238,7 +234,7 @@ function test_input($data) {
...
+	Good work
```
Comment (dilipchauhan14) [inline index.php]: Appreciation.

Comment (sammarth-kapse) [inline index.php]: You should get it.

**Pull request merged**